ed/idlpatches/SVG-void.idl.patch

```diff
@@ -0,0 +1,185 @@
+From a8fbc3ca92b35de70d2639bc4fbe902b97c30c17 Mon Sep 17 00:00:00 2001
+From: =?UTF-8?q?Philip=20J=C3=A4genstedt?= <philip@foolip.org>
+Date: Wed, 14 Apr 2021 09:11:50 +0200
+Subject: [PATCH] Fix void in SVG.idl
```

Comment:
Note that this is `ed/idlpatches/SVG-void.idl.patch` instead of merging with `ed/idlpatches/SVG.idl.patch`, on the expectation that this patch can soon be dropped while the existing one cannot.

Reply:
Actually I'm going to fold `svg-animations.idl.patch` into this, since it's from the same spec PR.